samples/react-timeline/src/webparts/timeline/components/TimelineActivity.tsx

```diff
@@ -0,0 +1,317 @@
+    let activityDate: string = moment(activity.acivityDate).format(this.state.dateFormat);
+
+    return (
+      <div>
+        {
+          this.props.canEdit &&
+          <div className={this.state.layout == "Vertical" ? `${styles.timelineAddVertical}` : `${styles.timelineAddHorizontal}`}>
+            <IconButton iconProps={addToIcon} title="Add Timeline Event" ariaLabel="Add Activity" className={styles.addToButton} onClick={this.createEvent} />
+          </div>
+        }
+
+        <Dialog type={DialogType.normal}
+          hidden={!this.state.showDeleteDialog}
+          title='Delete event?'
+          subText='Do you want to delete this event?'
+          isBlocking={true}
+          containerClassName={'ms-dialogMainOverride'}>
+          <DialogFooter>
+            <PrimaryButton onClick={() => { this.deleteEvent(this.state.selectedEvent); }} text="Yes" />
+            <DefaultButton onClick={() => { this.setState({ selectedEvent: null, showDeleteDialog: false }); }} text="No" />
+          </DialogFooter>
+        </Dialog>
+
+        <div className={this.state.layout == "Vertical" ? `${styles.timelineContentVertical}` : `${styles.timelineContentHorizontal}`}>
+          <div className={this.state.layout == "Vertical" ? `${styles.timelineRowVertical}` : `${styles.timelineRowHorizontal}`}>
+            {index % 2 == 1 &&
+              <div className={this.state.layout == "Vertical" ? `${styles.timelineColumnVertical}` : `${styles.timelineColumnHorizontal} ${styles.timelineDateHorizontalTop}`}>
+                <div className={styles.timelineDate}>
+                  <Text styles={helpfulTextStyles}>
+                    {activityDate}
+                  </Text>
+                </div>
+              </div>
+            }
+
+            <div className={this.state.layout == "Vertical" ? `${styles.timelineColumnVertical}` : `${styles.timelineColumnHorizontal}`}>
+              <Stack tokens={sectionStackTokens}>
+                {this.state.showEventDialog && (
+                  <TimelineEvent
+                    event={this.state.selectedEvent}
+                    panelMode={this.state.panelMode}
+                    onDissmissPanel={this.onDismissPanel}
+                    showPanel={this.state.showEventDialog}
+                    context={this.props.context}
+                    listName={this.props.listName}
+                  />
+                )}
+                <div className={styles.timelineCard}>
+                  <Card
+                    aria-label="Clickable horizontal card "
+                    horizontal
+                    tokens={cardTokens}
+                  >
+                    {
+                      showImage && activity.activityPictureUrl &&
+                      <Card.Item fill>
+                        <Image
+                          src={activity.activityPictureUrl ? activity.activityPictureUrl["Url"] : ''}
+                          alt="Placeholder image."
```

Comment:
Recommend using event's title instead of `placeholder image`. It would be a fantastic demonstration of accessibility support.